eng/common/scripts/Update-ChangeLog.ps1

```diff
@@ -80,40 +84,44 @@ if ($ChangeLogEntries.Contains($Version))
 $PresentVersionsSorted = [AzureEngSemanticVersion]::SortVersionStrings($ChangeLogEntries.Keys)
 $LatestVersion = $PresentVersionsSorted[0]
 
+LogDebug "The latest release note entry in the changelog is for version [$($LatestVersion)]"
+
 $LatestsSorted = [AzureEngSemanticVersion]::SortVersionStrings(@($LatestVersion, $Version))
 if ($LatestsSorted[0] -ne $Version) {
-    LogWarning "Passed Version [$Version] is older than the latestversion [$LatestVersion] in the changelog. Please use a more recent version."
+    LogWarning "Version [$Version] is older than the latestversion [$LatestVersion] in the changelog. Please use a more recent version."
     exit(0)
 }
 
-if ($ReplaceLatestEntry) 
+if ($ReplaceLatestEntryTitle) 
 {
+    $newChangeLogEntry = New-ChangeLogEntry -Version $Version -Status $ReleaseStatus -Content $ChangeLogEntries[$LatestVersion].ReleaseContent
+    LogDebug "Resetting latest entry title to [$($newChangeLogEntry.ReleaseTitle)]"
```

Comment:
Why LogDebug here as opposed to a standard log message?

Reply:
I could make it the standard output. Just the LogDebug is colored blue on DevOps which looks quite better, that's the only reason am using it here.

Reply:
Ok I guess LogDebug just didn't seem right in my head as I associated it with Write-Debug which will not write anything unless Debug preference is setup. We may want to think about the naming that function a little more.